lib/utils.coffee

```diff
@@ -194,5 +207,43 @@ globalRoot.extend = (hash1, hash2) ->
     hash1[key] = hash2[key]
   hash1
 
+# A simple cache. Entries used within an expiry period are retained (for one more expiry period), otherwise
+# they are discarded.
+class SimpleCache
+  # expiry: expiry time in milliseconds (default, one hour)
+  # entries: maximum number of entries
```

Comment:
> However, it allows access to all of them.

The code seems to indicate otherwise... It allows access to *any* `entries` of the `2 * entries`, perhaps?